src/pages/components/progress-bar/usage.mdx

```diff
@@ -0,0 +1,504 @@
+
+</Column>
+
+<Column colLg={4}>
+
+![Indeterminate progress bars move along a fixed track continually until the process is complete.](images/progress-bar-usage-3.gif)
+
+<Caption>
+  Indeterminate progress bars move along a fixed track continually until the
+  process is complete.
+</Caption>
+
+</Column>
+</Row>
+
+## Determinate progress bar
+
+Determinate progress bars are used when the loading progress can be tracked and
+translated into a percentage indicator which fills from 0% to 100% and never
+decrease in value. The entire animation moves from the left to the right – with
+no back and forth motion.
+
+Use a determinate progress bar when progress can be calculated against a
+specific goal. For example, when downloading a file of a known size.
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar uploading](images/progress-bar-usage-4.gif)
+
+</Column>
+</Row>
+
+## Indeterminate progress bar
+
+Once triggered, the loading bar will begin by moving along in constant speed,
+repeating over time while data is being gathered during the first stages of the
+load. This bar then changes to a determinate indicator (finishing from left to
+right) when enough information is gathered to end the load animation. The final
+determinate stage of the load is used to visually indicate the completion of the
+task to the user. In this scenario, a percentage indicator would not be present.
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar uploading](images/progress-bar-usage-5.gif)
+
+</Column>
+</Row>
+
+### Progress status
+
+It's best to give the user transparency about the process which usually leads to
+increased trust in the software or platform. However, it may not be possible to
+provide progress estimation in certain cases. If progress data is not available,
+an indeterminate progress bar can be used.
+
+As a more concrete set of examples, there are generally three different major
+progressions of a system operation: downloading, uploading, and local
+processing.
+
+#### Downloading
+
+For downloading actions it's usually possible to get a value of progress and
+update it in real time for a determinate progress bar. Depending on the
+information reported from a data source, time remaining could be estimated based
+on how much has been downloaded so far over a time period.
+
+#### Uploading
+
+Upload actions may be unable to report real time progress status depending on
+the data source and API configuration. However, always strive to provide users
+the determinate value whenever it's possible to inform them about the process.
+
+#### Data processing
+
+Data processing refers to manipulations of data to produce meaningful
+information. This can include currency exchange, file generation, data
+reformatting, or similar operations. Depending on the operation, time estimated
+or cycles to complete processing could be used to populate a determinate
+progress bar.
+
+## Formatting
+
+Determinate and indeterminate progress bars both have a label and an optional
```

Comment:
This information seems out of place in the general formatting section. I'd move it either to include it in the anatomy or content section.